### DISCUSSION TITLE
103

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#17] [General]
# 潮流周刊第103期 - 茶山的树

封面来源于周六去了一趟光明寺水库晒阳光，弄了一个露营毯睡了一觉，听着李志，很舒服的。

https://weekly.tw93.fun/posts/103-%E8%8C%B6%E5%B1%B1%E7%9A%84%E6%A0%91

**Comment by janebooom** [comment]
光明寺水库这张，拍的很好看👍

**Reply by tw93** [reply]
谢谢喜欢

**Comment by avezah** [comment]
这个主题太美了，简洁大方。

**Reply by tw93** [reply]
666  喜欢可以拿过去改吧改吧 自己用

**Comment by arstnei0** [comment]
真的很喜欢这种设计
有没有什么英文字体和 lxgw wenkai 像的呢
我现在用的是ysabeau
不可能英语网页也加载lxgw吧要死了

**Reply by tw93** [reply]
哈哈 我就是 也还行
